tests/components/abode/conftest.py

```diff
@@ -0,0 +1,22 @@
+"""Configuration for Abode tests."""
+import abodepy.helpers.constants as CONST
+import pytest
+
+from tests.common import load_fixture
+
+
+@pytest.fixture(autouse=True)
+def requests_mock_fixture(requests_mock):
+    """Fixture to provide a requests mocker."""
+    # Mocks the login response for abodepy.
+    requests_mock.post(CONST.LOGIN_URL, text=load_fixture("abode_login.json"))
+    # Mocks the oauth claims response for abodepy.
+    requests_mock.get(
+        CONST.OAUTH_TOKEN_URL, text=load_fixture("abode_oauth_claims.json")
```

Comment:
Instead of using `load_fixture` and having a separate JSON file with:
```
{
            "token_type": "Bearer",
            "access_token": "ohyeahthisisanoauthtoken",
            "expires_in": 3600,
        }
```
Is it acceptable to simply have this return the JSON above directly? That way I can simply delete fixtures/abode_oauth_claims.json.

Additionally, I think I'll do the same for the logout. I removed it in a previous PR but I have a feeling when each test finishes, it's making an actual post request for logout to the Abode servers which is obviously not ideal. Not entirely sure how to confirm this but just something I was thinking about.

Reply:
Since we have the other fixture files I think we should keep this in a json file too. If it was just one fixture with three lines we could have scrapped load_fixture etc.